### PR TITLE
blank volume support

### PR DESCRIPTION
**Pull request by giggsoff**
Support for blank volumes (with origin type VolumeContentOriginType_VCOT_BLANK).
For now we use raw format for them and expect maxvolumesize to have non-zero value

Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>

### CODE REVIEW

File: pkg/pillar/cmd/volumemgr/updatestatus.go [hunk 1]
```diff
@@ -414,17 +414,16 @@ func doUpdateVol(ctx *volumemgrContext, status *types.VolumeStatus) (bool, bool)
...
 			status.Key(), status.DisplayName, status.VolumeContentOriginType)
 		status.SetErrorWithSource(errStr,
 			types.VolumeStatus{}, time.Now())
-		changed = true
-		return changed, false
-	case zconfig.VolumeContentOriginType_VCOT_DOWNLOAD:
+		return true, false
+	}
+	changed := false
+	if status.VolumeContentOriginType == zconfig.VolumeContentOriginType_VCOT_DOWNLOAD {
```
Comment (eriknordmark) [inline pkg/pillar/cmd/volumemgr/updatestatus.go:426]: Any particular reason the case statement on VolumeContentOriginType doesn't work with these changes? Couldn't you just add the UNKNOWN to the case statement or is there some other subtle change here?

File: pkg/pillar/diskmetrics/diskmetrics.go [hunk 2]
```diff
@@ -56,6 +56,18 @@ func ResizeImg(log *base.LogObject, diskfile string, newsize uint64) error {
 	return nil
 }
 
+//CreateImg creates empty image file with defined format and size
+func CreateImg(log *base.LogObject, diskfile string, format string, size uint64) error {
+	output, err := base.Exec(log, "/usr/bin/qemu-img", "create", "-f", format, diskfile,
+		fmt.Sprintf("%d", size)).CombinedOutput()
```
Comment (eriknordmark) [inline pkg/pillar/diskmetrics/diskmetrics.go:62]: Strconv.Itoa or whatever it is called is nicer that a Sprintf like this.

File: pkg/pillar/cmd/volumemgr/updatestatus.go (hunk 1 above)
Comment (giggsoff) [reply pkg/pillar/cmd/volumemgr/updatestatus.go]: Yes, indeed, returned switch

File: pkg/pillar/diskmetrics/diskmetrics.go (hunk 2 above)
Comment (giggsoff) [reply pkg/pillar/diskmetrics/diskmetrics.go]: Moved to `strconv.FormatUint`

**Review by eriknordmark — APPROVED** [review]
LGTM

**Pull request merged**